config.py

```diff
@@ -48,9 +49,13 @@
 ################################################################################
 
 # TRACKER_LIST example:
-#TRACKER_LIST=[("tracker.hostname.tld", 5499),
-#              ("127.0.0.1", 5499)]
-TRACKER_LIST=[]
+TRACKER_LIST=[("tracker.preterhuman.net", 5499),
+              ("tracked.agent79.org", 5499),
+              ("hotline.duckdns.org", 5499),
+              ("tracked.stickytack.com", 5499),
+              ("tracked.nailbat.com", 5499),
+              ("hotline.ubersoft.org", 5499)]
+#TRACKER_LIST=[]
```

Comment:
best would be to have these but leave them commented out so that it works out of the box for servers that don't want to advertise themselves

Reply:
yeah, i didn't plan on changing this file. will revert